.github/ISSUE_TEMPLATE/01_general_issue_template.yml

```diff
@@ -0,0 +1,27 @@
+name: General
+description: File an issue that is not covered by other issue templates
+title: "[General]:"
+labels: [general]
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for taking the time to fill out this form!
+
+- type: input
+  id: contact
+  attributes:
+    label: Contact Details
+    description: How can we get in touch with you if we need more info?
+    placeholder: eg. email@example.com
+  validations:
+    required: false
+
```

Comment:
This is actually the place where I'd expect to (optionally) chose a layer and functional cluster, e.g. in cases where I'm proposing a new building block.

Reply:
The proposal of new BB should not happen via Github but through other channels such as meetings or MS Forms.